packages/protocol/utils/airdrop/buildMerkleTree.ts

```diff
@@ -0,0 +1,83 @@
+import { MerkleTree } from "merkletreejs/dist/MerkleTree";
+import { claimList } from "./airdrop_db/claimList";
```

Comment:
custom json file input from cmd

Reply:
This is the only pending issue @adaki2004